pkg/config/generate_test.go

```diff
@@ -112,3 +113,54 @@ func testRequiredFields(cs *api.OpenShiftManagedCluster, t *testing.T) {
 	assert(c.NodeBootstrapKubeconfig != nil, "NodeBootstrapKubeconfig")
 	assert(c.AzureClusterReaderKubeconfig != nil, "AzureClusterReaderKubeconfig")
 }
+
+func TestGenerateUpdateCertRegen(t *testing.T) {
```

Comment:
This test will run pretty slowly.  Can't you just poke some pairs of certificates into certEqual and check that the return value is as expected?

Reply:
go caches the test result, so not a big issue. And it shows how our other functions work in combined. I can do this, but I think this allows to test full flow too. WDYT?

Reply:
I disagree.